2/contrib/jenkins/install-plugins.sh

```diff
@@ -0,0 +1,344 @@
+#! /bin/bash -eu
+#
+# Originally copied from https://github.com/jenkinsci/docker
+# You can set JENKINS_UC to change the default URL to Jenkins update center
+#
+# Usage:
+#
+# FROM openshift/jenkins-2-centos7
+# COPY plugins.txt /plugins.txt
+# RUN /usr/local/bin/install-plugins.sh /plugins.txt
+#
+# The format of 'plugins.txt. is:
+#
+# pluginId:pluginVersion
+
+set -o pipefail
+
+# BEGIN - From https://raw.githubusercontent.com/jenkinsci/docker/master/jenkins-support
```

Comment:
So I'm assuming that jenkinsci updated their plugin install script since we originally created the V1 version, and that is why they differ ... that correct?